### ISSUE TITLE
jspm 0.17 beta 13 moves dependencies between peer and dev during "jspm install"

### ISSUE THREAD

**Issue by jithin1987** [jspm/jspm-cli#1742]
This happens during each `jspm install` without pre existing `jspm_packages` folder. For my `package.json` this is the diff between what I have on my github and what jspm did when running `jspm install` on a build box.

```
diff -u  package.json package.new.json
--- package.json        2016-04-20 10:47:32.000000000 -0700
+++ package.new.json    2016-04-20 14:28:41.000000000 -0700
@@ -1,3 +1,4 @@
+
 {
   "name": "dashboard",
   "description": "Dashboard",
@@ -118,11 +119,13 @@
     "devDependencies": {
       "babel-plugin-transform-class-properties": "npm:babel-plugin-transform-class-properties@^6.6.0",
       "babel-plugin-transform-react-jsx": "npm:babel-plugin-transform-react-jsx@^6.7.5",
+      "child_process": "github:jspm/nodelibs-child_process@^0.2.0-alpha",
       "clean-css": "npm:clean-css@^3.4.12",
       "core-js": "npm:core-js@^2.2.2",
       "fs": "github:jspm/nodelibs-fs@^0.2.0-alpha",
       "module": "github:jspm/nodelibs-module@^0.2.0-alpha",
       "net": "github:jspm/nodelibs-net@^0.2.0-alpha",
+      "path": "github:jspm/nodelibs-path@^0.2.0-alpha",
       "plugin-babel": "npm:systemjs-plugin-babel@^0.0.9",
       "process": "github:jspm/nodelibs-process@^0.2.0-alpha",
       "tty": "github:jspm/nodelibs-tty@^0.2.0-alpha"
@@ -131,13 +134,11 @@
       "assert": "github:jspm/nodelibs-assert@^0.2.0-alpha",
       "buffer": "github:jspm/nodelibs-buffer@^0.2.0-alpha",
       "chart.js": "npm:chart.js@^1.0.0",
-      "child_process": "github:jspm/nodelibs-child_process@^0.2.0-alpha",
       "domain": "github:jspm/nodelibs-domain@^0.2.0-alpha",
       "events": "github:jspm/nodelibs-events@^0.2.0-alpha",
       "http": "github:jspm/nodelibs-http@^0.2.0-alpha",
       "https": "github:jspm/nodelibs-https@^0.2.0-alpha",
       "os": "github:jspm/nodelibs-os@^0.2.0-alpha",
-      "path": "github:jspm/nodelibs-path@^0.2.0-alpha",
       "redux": "npm:redux@^3.0.0",
       "stream": "github:jspm/nodelibs-stream@^0.2.0-alpha",
       "string_decoder": "github:jspm/nodelibs-string_decoder@^0.2.0-alpha",
@@ -196,3 +197,4 @@
     ]
   }
 }
+
```

Why is jspm modifying `package.json` when running on a different machine ?

**Comment by jithin1987** [comment]
Also in package.json config what is the difference between jspm.dependencies / jspm.peerDependencies ? 
When I run `jspm install --peer` I see that everything from `dependencies` move to `peerDependencies` and if I do `jspm install --dev` everything from `dependencies` move to `devDependencies`

**Comment by Palid** [comment]
I had exactly the same experience on three different developer machines. Each time developer did `npm install` the packages danced around.

**Comment by guybedford** [comment]
Sure thanks, will look into this further.

**Comment by guybedford** [comment]
I'm still unable to replicate this at all, can someone share some exact steps that show the switching behaviour?

**Comment by lookfirst** [comment]
For me it was just `jspm install` on different machines. All running same version of node and jspm. It is pretty random.

**Comment by jithin1987** [comment]
Also make sure `jspm_packages` folder does not exist prior to `jspm install`

**Comment by guybedford** [comment]
Fixed in https://github.com/jspm/jspm-cli/commit/a23797237817d9442fc5911a2ef4fdb631a3fdb4.

**Comment by guybedford** [comment]
Released in jspm 0.17.0-beta.15.
